samples/net/civetweb/http_server/prj.conf

```diff
@@ -30,3 +30,7 @@ CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"
 
 # logging
 CONFIG_NET_LOG=y
+
+# Do not include mbedtls via this option as civetweb does not
+# work properly with mbedtls.
+CONFIG_NET_TCP_ISN_RFC6528=n
```

Comment:
Could this be checked by a `depends on !CONFIG_NET_TCP_ISN_RFC6528` in the `CIVETWEB` Kconfig declaration, so this isn't only checked for these two samples?

Reply:
> Could this be checked by a `depends on !CONFIG_NET_TCP_ISN_RFC6528` in the `CIVETWEB` Kconfig declaration

Sure, I can send a separate PR for that.

Reply:
See #33215